.changeset/stale-llamas-cheer.md

```diff
@@ -0,0 +1,5 @@
+---
+"@ledgerhq/live-common": minor
+---
+
+upgrade dependencies
```

Comment:
this is too vague to understand in a changeset. please precise exactly which dependencies 🙏